recruitment/developer-API-task.md

```diff
@@ -0,0 +1,34 @@
+###Web Development Task
+
+You've been sent this task as we would love to get an idea of the way you work.
+
+At Holiday Extras we use a variety of different technologies and we don't expect you to be aware of all of them! We've designed this task with this in mind. It's worth noting that everything described below is a rough guideline of what we're looking for so don't feel you have to follow it exactly.
+
+
+#####Your task is to create an api to manage a user persistence layer.
+
+We would expect this task to take a few hours, however there is no strict time limit and you won't be judged on how long it took you to complete. Please find a few pointers below:
+
+* Your solution must expose a user model and it's reasonable to expect that an individual user would have the following attributes:
+
+	**id** - *A unique user id*  
+	**email** - *A users email address*  
+	**forename** - *A users first name*  
+	**surname** - *A users last name*  
+	**created** - *The date and time the user was added*
+
+* It must have the ability to persist user information for at least the lifetime of the test.
+
+* You api must expose functionality to create, read, update and delete (CRUD) models.
+
+Although the main outcomes of the task are listed above, if you feel like you want to go that extra mile and show us what you're capable of, here is a list of potential enhancements that we have come up with:
+
+
+* How your api is to be consumed (a custom interface or something like Google Chromes "Postman" or Swagger).
+* Use of an industry standard data exchange format.
+
+Alternatively if you can think of any other features that you feel would further enhance your api, then we'd love to see what you can come up with!
+
+When you feel you've completed the task we would appreciate it if you could publish it online so we are in a position to be able to review it. We recommend that you upload your code to a Github/Bitbucket repository, however it's fine if you'd rather send it as an attachment in an email.
```

Comment:
We can maybe suggest [GitLab](https://about.gitlab.com/) as another alternative.